README.md

```diff
@@ -1,74 +1,105 @@
+```
 
-Purge any existing developer chain state:
+### Installing a release build
+This will install the executable `subsocial-node` to your `~/.cargo/bin` folder, which you would normally have in your `$PATH` environment.
 
 ```bash
-./target/release/subsocial-node purge-chain --dev
+cargo install --path ./
 ```
 
-Start a development chain with:
+Now you can run
 
 ```bash
-./target/release/subsocial-node --dev
+subsocial-node
 ```
 
-Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1 cargo run -- --dev`.
+## Building from Docker
 
-### Multi-Node Local Testnet
+### Easiest start
+To start local Subsocial Node (you should have docker-compose):
 
-If you want to see the multi-node consensus algorithm in action locally, then you can create a local testnet with two validator nodes for Alice and Bob, who are the initial authorities of the genesis chain that have been endowed with testnet units.
+```
+cd docker/
+docker-compose up -d
+```
 
-Optionally, give each node a name and expose them so they are listed on the Polkadot [telemetry site](https://telemetry.polkadot.io/#/Local%20Testnet).
+### Start with your own  parameters
 
-You'll need two terminal windows open.
+```
+docker run -p 9944:9944 dappforce/subsocial-node:latest ./subsocial-node [flags] [options]
+```
+* Don't forget `--ws-external` flag, if you want your node to be visible no only within the container.
+
+### Connect to our network with Docker
+If you're interested in launching your Node that will be connected to Subsocial Testnet, you should run the next command:
+```
+docker run -p 30333:30333 -p 9933:9933 -v <YourPath>:/data dappforce/subsocial-node:df-v2 ./subsocial-node -d /data --name <NodeName> --pruning archive --bootnodes /ip4/167.172.104.62/tcp/30333/p2p/QmfHoYER3gPchRvHSTzyFZ9hhvQG3oP9jgcQEbfbm2x9LR /ip4/167.172.104.62/tcp/30334/p2p/QmVcp8pBKtKcwWmWcrARZcdLxW9E1EzfU9JwvgiSKu4How
+```
+, where:
+`<YourPath>` - should be replaced with a full path, where node's data will be stored;
+`<NodeName>` - node's name you wish to be shown on a [telemetry](https://telemetry.polkadot.io/#list/Subsocial%20Barracuda%20Testnet) webpage.
+Be patient, current active bootnodes' URIs are:
+- Alice: `/ip4/167.172.104.62/tcp/30333/p2p/QmfHoYER3gPchRvHSTzyFZ9hhvQG3oP9jgcQEbfbm2x9LR`
+- Bob: `/ip4/167.172.104.62/tcp/30334/p2p/QmVcp8pBKtKcwWmWcrARZcdLxW9E1EzfU9JwvgiSKu4How`
 
-We'll start Alice's substrate node first on default TCP port 30333 with her chain database stored locally at `/tmp/alice`. The bootnode ID of her node is `QmRpheLN4JWdAnY7HGJfWFNbfkQCb6tFf4vvA6hgjMZKrR`, which is generated from the `--node-key` value that we specify below:
+### Build your own image
+If you want to build docker image from your local repository (it takes a while...), run the following command from the root of the repository:
```

Comment:
```suggestion
If you want to build a docker image from your local repository (it will take some time...), run the following command from the root of your repository:
```